packages/sw-cacheable-response/test/sw/cacheable-response.js

```diff
@@ -0,0 +1,108 @@
+importScripts(
+  '/node_modules/mocha/mocha.js',
+  '/node_modules/chai/chai.js',
+  '/node_modules/sw-testing-helpers/build/browser/mocha-utils.js',
+  '/packages/sw-cacheable-response/build/sw-cacheable-response.js'
+);
+
+const expect = self.chai.expect;
+mocha.setup({
+  ui: 'bdd',
+  reporter: null,
+});
+
+describe('Test of the CacheableResponse class', function() {
+  const validStatus = 418;
```

Comment:
I'd rather you changed to VALID_HEADERS - just makes reading and following the tests easier